src/js/coblocks-lightbox.js

```diff
@@ -3,7 +3,7 @@
 
 	const lightboxModals = document.getElementsByClassName( 'has-lightbox' );
 
-	Array.from( lightboxModals ).forEach( ( lightbox, index ) => {
+	Array.from( lightboxModals ).forEach( function( lightbox, index ) {
```

Comment:
Why are we changing fat arrows to function? Is this because of a problem with `this` or this is to unify the codebase?

Reply:
This was to unify the codebase specifically. I realized that this was the only `forEach` method using arrow syntax, so I made that change to unify to use of `forEach`.